modal/runner/engines/vllm.py

```diff
@@ -68,9 +70,11 @@ async def generate(self, payload: CompletionPayload, params):
         try:
             import time
 
-            results_generator = self.engine.generate(
-                payload.prompt, params, payload.id
-            )
+            tags = {"model": self.engine_args.model}
+            with timer("engine.generate()", tags=tags):
+                results_generator = self.engine.generate(
+                    payload.prompt, params, payload.id
+                )
```

Comment:
have a suspicion that vllm is lazy loading the weights into memory, so it might be happening here vs inside the constructor.  this will help inform whether thats the case